### PR TITLE
Highlight the currently 'dragged-over' resource and time

### PR DESCRIPTION
**Pull request by benbarnett**
To help resource managers know they're dragging and dropping
events into the correct areas.

aomnmza5lo

### CODE REVIEW

**Comment by benbarnett** [comment]
@AndrewVos @Guntrisoft decided to add the whole row highlight thing. 

Any thoughts?

**Comment by AndrewVos** [comment]
I love it. Does it work when there's a lot more guiders?

**Comment by benbarnett** [comment]
@AndrewVos yep, seems to when I try locally. It's fairly optimised as the position recalculation only happens after each 'snap' of the dragging basically.

File: app/assets/stylesheets/components/_calendars.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@ $calendar-cancelled-background: $color-guardsman-red;
 $calendar-moved-background: $color-green;
 $calendar-filter-box-shadow: transparentize($color-black, .825);
 $calendar-filter-arrow-border: transparentize($color-black, .8);
+$calendar-dragging-highlight: $color-givry;
```
Comment (AndrewVos) [inline app/assets/stylesheets/components/_calendars.scss]: So today I learned that givry is a colour

**Comment by AndrewVos** [comment]
LGTM

**Pull request merged**